packages/owl-bot/src/copy-code.ts

```diff
@@ -325,7 +325,11 @@ export async function copyCode(
   const cmd = newCmd(logger);
   const sourceDir = toLocalRepo(sourceRepo, workDir, logger);
   // Check out the specific hash we want to copy from.
-  if (sourceCommitHash) {
+  if ('none' === sourceCommitHash) {
+    // User is running copy-code from command line.  The path specified by
+    // sourceRepo is not a repo.  It's a bazel-bin directory, so there's
+    // no corresponding commit hash, and that's ok.
+  } else if (sourceCommitHash) {
     cmd(`git checkout ${sourceCommitHash}`, {cwd: sourceDir});
   } else {
```

Comment:
What is the third alternative here? Perhaps helpful to add a comment so as to understand what to expect.

Reply:
Done.